CMakeLists.txt

```diff
@@ -104,6 +104,8 @@ option(F90CHARM            "Build f90charm" OFF)
 option(PEDANTIC            "Build with -pedantic" OFF)
 option(BUILD_CUDA          "Build with CUDA support" OFF)
 
```

Comment:
Is this blank line intentional?

Reply:
Somewhat. The intention here was to separate the "official" options mentioned in the help of the old ./build script from the undocumented expert options.